src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,26 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+import { TodoWUser } from '../../types/TodoWUser';
+import { UserInfo } from '../UserInfo';
+
+export type Props = {
```

Comment:
```suggestion
export interface Props = {
```
why need to export props type from the component?